plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/RemoveFileEntry.java

```diff
@@ -43,6 +67,24 @@ public String getPath()
         return path;
     }
 
+    @Nullable
+    @JsonProperty
+    @Deprecated // required for JSON serialization; getCanonicalPartitionValues should be used in code instead.
+    public Map<String, String> getPartitionValues()
+    {
+        return partitionValues;
+    }
+
+    /**
+     * @return the original key and canonical value. The value returns {@code Optional.empty()} when it's null or empty string.
+     */
+    @Nullable
+    @JsonIgnore
+    public Map<String, Optional<String>> getCanonicalPartitionValues()
```

Comment:
This method is unused and marked as `JsonIgnore` annotation. Is it possible to remove this method and the field?